java/pending_code_changes.txt

```diff
@@ -1 +1,3 @@
-
+Code changes:
+ - Made normalizeDiallableCharsOnly() API in PhoneNumberUtil public. This
```

Comment:
s/API in PhoneNumberUtil public./in PhoneNumberUtil public in Java API./

And I'd mention that the method was added to JS too.